src/os/bluestore/BlueFS.cc

```diff
@@ -480,6 +480,7 @@ void BlueFS::_update_logger_stats()
 }
 
 int BlueFS::add_block_device(unsigned id, const string& path, bool trim,
+                             const char* type,
```

Comment:
Instead of passing the type from outside IMO it's better to assign the name locally in the switch below given the provided id.